modules/examples/routeguide/client/src/main/scala/Client.scala

```diff
@@ -0,0 +1,6 @@
+package routeguide.client
```

Comment:
Let's looking for a different package naming for all the examples in order to have unified all examples with the same prefix.

For example, `package examples.routeguide.client``

What do you think?